packages/core/CHANGELOG.md

```diff
@@ -1,5 +1,54 @@
 # xstate
 
+## 4.16.0
```

Comment:
Please do not merge master to feature branches targeting next before you merge master to next itself 🙏  it makes reviewing much harder

Reply:
Ah, oops, sorry!